src/static/js/config/aria-states-config.js

```diff
@@ -0,0 +1,79 @@
+/* ==========================================================================
+   Aria State
+
+   Code copied from the following with minor modifications:
+
+   - https://github.com/IBM-Watson/a11y.js
+     Copyright (c) 2014 IBM
```

Comment:
Could use an MIT license entry in https://github.com/cfpb/cfgov-refresh/blob/flapjack/TERMS.md#exceptions